libraries/load-themed-styles/src/index.ts

```diff
@@ -397,6 +402,13 @@ function registerStyles(styleArray: ThemableArray): void {
   _themeState.perf.count++;
   head.appendChild(styleElement);
 
+  var ev: ICustomEvent<{ newStyle: HTMLStyleElement}> = document.createEvent('HTMLEvents');
+  ev.initEvent("load-themed-styles-added", true /* bubbleEvent */, false /* cancelable */);
```

Comment:
if we just call the event "styleinsert", which may be a little more analogous to "click" or "focus" or "mousedown", we can keep this general and possibly use the same event for merge-styles.

if style elements are inserted, this would be fired. But if rules are being inserted using insertRule, we might need "ruleinsert" later. This doesn't apply in this library, but just trying to think how a customer could leverage these scenarios in a consistent way that isn't super implementation specific.

Reply:
what do you think? If you want to keep as is to keep it scoped, I'm ok. But maybe this is a good opportunity to snap to a less proprietary way to do this.